os-autoinst-obs-auto-submit

```diff
@@ -7,7 +7,7 @@ src_project=${src_project:-devel:openQA}
 dst_project=${dst_project:-${src_project}:tested}
 staging_project=${staging_project:-${src_project}:testing}
 submit_target="${submit_target:-"openSUSE:Factory"}"
-submit_target_escaped=$(echo -n "$submit_target" | sed -e 's|\:|_|g') # avoid, e.g. "repoid 'openSUSE:Factory' is illegal"
+submit_target_supplemental="${submit_target_list:-}"
```

Comment:
Why not just

```suggestion
submit_target_extra="${submit_target_extra:-}"
```

Reply:
The "extra" was fine, it just means "extra" to the base target which is already there